NEWS.md

```diff
@@ -4,6 +4,8 @@
 
 * Exported the `qenv` class from the package.
 * The `@code` field in the `qenv` class now holds `character`, not `expression`.
+* The `get_code` method  returns a single concatenated string of the code.
+* Removed internal `format_expression` function.
```

Comment:
I don't think removing `format_expression` should be added to `NEWS.md`. It's not a user-facing function and it will just clutter the actual news.
```suggestion
```